### PR TITLE
Adding debug flags and tools

### PR DESCRIPTION
**Pull request by msooseth**
## Description
Allows to give `-f debug` flag to `cabal` so it builds with extensive debug information and eventlog enabled. We now also include `eventlog2html` as part of our cabal. This should aid in debugging in the future.

## Checklist

- [x] tested locally
- [ ] added automated tests
- [ ] updated the docs
- [x] updated the changelog

### CODE REVIEW

File: hevm.cabal [hunk 1]
```diff
@@ -241,7 +253,8 @@ executable hevm
     vty,
     stm,
     spawn,
-    optics-core
+    optics-core,
+    eventlog2html
```
Comment (d-xo) [inline hevm.cabal:257]: Does this actually need to be in the cabal file? Can it go in the nix shell instead? I would prefer that it's not listed as a direct dep of the exe unless actually required...

Comment (msooseth) [reply hevm.cabal]: Yeah, good point. Moved to nix :)

**Comment by msooseth** [comment]
Annoyingly, some random bug got exposed via fuzzing :) I'm tracking it down.

**Review by d-xo — APPROVED** [review]
lgtm

**Review by arcz — APPROVED** [review]

**Pull request merged**